sdk/python/feast/feature_store.py

```diff
@@ -950,7 +951,9 @@ def apply(
                     validation_references.name, project=self.project, commit=False
                 )
 
-        tables_to_delete: List[FeatureView] = views_to_delete + sfvs_to_delete if not partial else []  # type: ignore
+        tables_to_delete: List[FeatureView] = (
+            views_to_delete + sfvs_to_delete if not partial else []
+        )  # type: ignore
```

Comment:
This is change created by `ruff format` raises a type check error with `mypy`: moving the annotation one line above to fix the issue